docker/build-tool/musl/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM rustembedded/cross:x86_64-unknown-linux-musl
+COPY ./openssl.sh /
```

Comment:
I really don't like the idea to compile openssl.

How about using `alpine` directly? https://github.com/datafuselabs/databend/pull/4536

Reply:
re-invent the alpine based build-kit is not easy, we could consider separate them into smaller tasks and iteratively replace current cross utilizations